pkg/inject/report.go

```diff
@@ -142,6 +150,24 @@ func checkUDPPorts(t *v1.PodSpec) bool {
 	return false
 }
 
+func checkNodeOS(t *v1.PodSpec) bool {
+	// Check any pods run a windows image, which is currently unsupported
+	//check the NodeSelector
+	osimage := t.NodeSelector["kubernetes.io/os"]
+	if strings.Contains(strings.ToLower(osimage), "windows") {
+		return true
+	}
+	//check if Node has been tainted
+	for _, toleration := range t.Tolerations {
```

Comment:
From what I have been reading [here](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/), Looks like they allow you to get repelled from specific nodes from being scheduled? Did I understand something wrong?

Reply:
no, you're right. I need to set this to look for taints may be, and set a tolerant... or something of that sort.